metrics/src/tests/stresstest.cpp

```diff
@@ -137,7 +138,7 @@ TEST(StressTest, test_stress)
             new Hammer(metrics, loadTypes, threadPool)));
     }
     LOG(info, "Waiting to let loadgivers hammer a while");
-    FastOS_Thread::Sleep(5 * 1000);
+    std::this_thread::sleep_for(std::chrono::milliseconds(5 * 1000));
```

Comment:
Consider simplifying to `std::this_thread::sleep_for(5s)`

Reply:
Fixed